### PR TITLE
ib_cni_ci scripts added

### PR DESCRIPTION
**Pull request by lennybe**

### CODE REVIEW

File: ib_cni_install.sh [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+echo "Get CPU architechture"
+export ARCH="amd"
+if [[ $(uname -a) == *"ppc"* ]]; then
+   export ARCH="ppc"
+fi
+
+function configure_multus {
+    echo "Configure Multus"
+
+    wget https://raw.githubusercontent.com/intel/multus-cni/master/images/multus-daemonset.yml -O $ARTIFACTS/multus-daemonset.yml
```
Comment (moshe010) [inline ib_cni_install.sh]: why it master and not per branch?

File: ib_cni_install.sh [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+        kubectl version
+    elif [ ${KUBERNETES_VERSION} == 'master' ]; then
+        git clone -b ${KUBERNETES_BRANCH} --single-branch --depth=1  https://github.com/kubernetes/kubernetes
+        cd kubernetes/
+        git show --summary
+        make
+        mv ./_output/local/go/bin/kubectl /usr/local/bin/kubectl
+    else
+        curl -LO https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_VERSION}/bin/linux/${ARCH}64/kubectl
+        #mv ./kubectl /usr/local/bin/kubectl
```
Comment (moshe010) [inline ib_cni_install.sh]: remove commet

File: ib_cni_test.sh [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+export LOGDIR=$WORKSPACE/logs
+export ARTIFACTS=$WORKSPACE/artifacts
+
+export GOROOT=${GOROOT:-/usr/local/go}
+export GOPATH=${WORKSPACE}
+export PATH=/usr/local/go/bin/:$GOPATH/src/k8s.io/kubernetes/third_party/etcd:$PATH
+export TIMEOUT=${TIMEOUT:-300}
+
+export POLL_INTERVAL=${POLL_INTERVAL:-10}
+export NETWORK=${NETWORK:-'192.168'}
```
Comment (moshe010) [inline ib_cni_test.sh]: what is "192.168" is that a network?

File: ib_cni_test.sh [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+export KUBECONFIG=${KUBECONFIG:-/var/run/kubernetes/admin.kubeconfig}
+pushd $WORKSPACE
+
+
+function pod_create {
+    cd $ARTIFACTS
+    wget https://raw.githubusercontent.com/Mellanox/k8s-rdma-shared-dev-plugin/master/example/test-hca-pod.yaml -O $ARTIFACTS/test-hca-pod.yaml
```
Comment (moshe010) [inline ib_cni_test.sh]: why master and not branch

File: ib_cni_test.sh [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+        echo "Failed to recognize net1 as infiniband interface with $NETWORK"
+    else
+        echo "Passed to recognize net1 as infiniband interface with $NETWORK"
+    fi
+    return $status
+}
+
+
+pod_create
+test_pod
```
Comment (moshe010) [inline ib_cni_test.sh]: please extend the test to create 2 pods and run ib_write_bw. Mamduh has the commands

**Review by moshe010 — CHANGES_REQUESTED** [review]
change the name of repo to ipoib_cni_ci and all other places

File: ib_cni_test.sh (hunk 3 above)
Comment (lennybe) [reply ib_cni_test.sh]: yes. I need to see that it on the interface inside.

File: ib_cni_install.sh [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+    fi
+
+    \cp bin/* $CNI_BIN_DIR/
+    popd
+
+    echo "Download and install kubectl"
+    rm -f ./kubectl /usr/local/bin/kubectl
+    if [ ${KUBERNETES_VERSION} == 'latest_stable' ]; then
+        export KUBERNETES_VERSION=$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)
+        curl https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_VERSION}/bin/linux/${ARCH}64/kubectl -o /usr/local/bin/kubectl
```
Comment (moshe010) [inline ib_cni_install.sh]: what ${ARCH}64 can't it all be in ${ARCH}

File: ib_cni_install.sh (hunk 1 above)
Comment (lennybe) [reply ib_cni_install.sh]: fixed

File: ib_cni_install.sh (hunk 2 above)
Comment (lennybe) [reply ib_cni_install.sh]: fixed

File: ib_cni_test.sh (hunk 5 above)
Comment (lennybe) [reply ib_cni_test.sh]: done

**Review by moshe010 — APPROVED** [review]

**Pull request merged**